src/shared-resize-observer.ts

```diff
@@ -0,0 +1,111 @@
+import { ResizeObserver as Polyfill } from '@juggle/resize-observer';
```

Comment:
yay, this will help x-browser tremendously

Reply:
I added this after I saw your MR come through to fix the old Safari issue 😄